spyderlib/plugins/shortcuts.py

```diff
@@ -807,6 +816,22 @@ def apply_settings(self, options):
         self.table.save_shortcuts()
         self.main.apply_shortcuts()
 
+    def load_button_clicked(self, change):
+        selected = self.combo_keybindings.currentText().lower()
+        reset_shortcuts()

```

Comment:
Yes that is what I told @blink1073, as soon as the user changes the defaults, a `custom` option should appear in the combobox and those shortcuts should be saved for later reuse...